src/ray/core_worker/reference_count.h

```diff
@@ -25,13 +27,33 @@
 #include "ray/rpc/worker/core_worker_client.h"
 #include "ray/util/logging.h"
 
-#include <boost/bind.hpp>
-
 namespace ray {
 
+// Interface for mocking.
```

Comment:
It is used for unit testing inside `ActorManager`. This is useful because I can mock the whole class in this way. I think we should start making interface for all classes, so that we can do unit testing more easily.